internal/installercache/installercache_test.go

```diff
@@ -53,60 +59,81 @@ var _ = Describe("release event", func() {
 			"cached", r.cached,
 			"extract_duration", r.extractDuration,
 		).Times(1)
-		r.Cleanup(ctx)
+		err = r.Cleanup(ctx)
+		Expect(err).To(BeNil())
```

Comment:
Nit, but I prefer `Expect(r.Cleanup(ctx)).To(Succeed())` for these cases.

Reply:
is better, one liner also